nav2_planner/src/planner_server.cpp

```diff
@@ -461,26 +461,52 @@ PlannerServer::computePlan()
       return;
     }
 
-    result->path = getPlan(start, goal_pose, goal->planner_id);
+    try
```

Comment:
There's already a try wrapping this full thing, I think you just needed to add the new catches before the default one